_config.yml

```diff
@@ -35,9 +35,9 @@ tierthreeplan: Tiny Cloud Professional Plan
 enterpriseplan: Tiny Custom Plans
 ## common tiny.cloud URLs
 supporturl: https://support.tiny.cloud
-plugindirectory: https://www.tiny.cloud/apps/
+plugindirectory: https://www.tiny.cloud/tinymce/features/
 gettiny: https://www.tiny.cloud/get-tiny
-accountpageurl: https://apps.tiny.cloud/my-account
+accountpageurl: https://www.tiny.cloud/my-account
 accountsignup: https://www.tiny.cloud/auth/signup
 productpages: https://apps.tiny.cloud/products
```

Comment:
would this one day need to be https://www.tiny.cloud/products/

Reply:
hopefully. But for now that is correct.